blog/_posts/2020-11-06-hybrid-cloud-with-inlets.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: Hybrid Cloud with inlets
+description: Learn how to connect on-premises services and cloud services with inlets &reg;
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud 
+author_img: jsiebens
+date: 06-11-2020
+---
+
+Learn how to connect on-premises services and cloud services with inlets &reg;
+
+## Hybrid Cloud?
+
+When people talk about Hybrid Cloud, they talk about a way to combine a private cloud or on-premises service with one or more public cloud services. A hybrid cloud strategy can give a huge benefit for your business by moving workloads to a public cloud, leveraging the flexibility and robustness of managed services, while keeping sensitive data on a private cloud or local data center.
```

Comment:
When people talk about "Hybrid Cloud" <- quote the term?

Which people?

How about using an authority as your definition instead? (Quoting Wikipedia)

https://en.wikipedia.org/wiki/Cloud_computing#Hybrid_cloud